apps.toml

```diff
@@ -1275,17 +1275,6 @@ state = "working"
 subtags = [ "backup" ]
 url = "https://github.com/YunoHost-Apps/garage_ynh"
 
-[garradin]
-added_date = 1674232499 # 2023/01/20
-antifeatures = [ "deprecated-software", "replaced-by-another-app" ]
```

Comment:
maybe still keep it as deprecated for one or two months?

Reply:
eeeh we could even have a special message in the diagnostic for apps in the graveyard